docs/src/components/version-switcher.tsx

```diff
@@ -49,7 +52,7 @@ export default function VersionSwitcher(): JSX.Element {
         mobile={windowSize === 'mobile'}
         items={versions.map(({ label, url }) => ({
           label,
-          to: url + location.pathname + location.hash,
+          to: new URL(location.pathname + location.search + location.hash, url).href,
```

Comment:
```suggestion
          to: new URL(location.pathname + location.search + location.hash, url).href,
          className: label === activeLabel ? "dropdown__link--active menu__link--active" : "",
```

@jrasm91 if I do this and change `label` to `activeLabel` in Line 7 and Line 51 then it'll highlight the active version in the dropdown.

Reply:
For reference: `dropdown__link--active` is for desktop and `menu__link--active` is for mobile.

Reply:
This can be part of a future PR though. This PR is focused on fixing the current path logic and stuff.